tests/integration/conftest.py

```diff
@@ -374,3 +374,124 @@ def create_nodebalancer_with_default_conf():
     res_arr = result.split(",")
     nodebalancer_id = res_arr[0]
     delete_target_id(target="nodebalancers", id=nodebalancer_id)
+
+
+@pytest.fixture
+def test_vpc_wo_subnet():
+    region = get_regions_with_vpcs_capabilties()[0]
+
+    label = str(time.time_ns()) + "label"
+
+    vpc_id = (
+        exec_test_command(
+            [
+                "linode-cli",
+                "vpcs",
+                "create",
+                "--label",
+                label,
+                "--region",
+                region,
+                "--no-headers",
+                "--text",
+                "--format=id",
+            ]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+
+    yield vpc_id
+
+    delete_target_id(target="vpcs", id=vpc_id)
+
+
+@pytest.fixture
+def test_vpc_w_subnet():
+    region = get_regions_with_vpcs_capabilties()[0]
+
+    vpc_label = str(time.time_ns()) + "label"
+
+    subnet_label = str(time.time_ns()) + "label"
+
+    vpc_id = (
+        exec_test_command(
+            [
+                "linode-cli",
+                "vpcs",
+                "create",
+                "--label",
+                vpc_label,
+                "--region",
+                region,
+                "--subnets.ipv4",
+                "10.0.0.0/24",
+                "--subnets.label",
+                subnet_label,
+                "--no-headers",
+                "--text",
+                "--format=id",
+            ]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+
+    yield vpc_id
+
+    delete_target_id(target="vpcs", id=vpc_id)
+
+
+@pytest.fixture
+def test_subnet(test_vpc_wo_subnet):
+    vpc_id = test_vpc_wo_subnet
+    subnet_label = str(time.time_ns()) + "label"
+    res = (
+        exec_test_command(
+            [
+                "linode-cli",
+                "vpcs",
+                "subnet-create",
+                "--label",
+                subnet_label,
+                "--ipv4",
+                "10.0.0.0/24",
+                vpc_id,
+                "--text",
+                "--no-headers",
+                "--delimiter=,",
+            ]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+
+    yield res, subnet_label
+
+
+def get_regions_with_vpcs_capabilties():
+    regions = (
+        exec_test_command(
+            [
+                "linode-cli",
+                "regions",
+                "ls",
+                "--text",
+                "--no-headers",
+                "--format=id,capabilities",
+            ]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+
+    regions = regions.split("\n")
+
+    regions_with_vpc = []
+
+    for region in regions:
+        if "VPCs" in region:
+            region_name = region.split()[0]  # Extract the region name
+            regions_with_vpc.append(region_name)
+
+    return regions_with_vpc
```

Comment:
Can we make it a global fixture and support getting regions with arbitrary capabilities? Similar to what we have in Terraform provider or linodego.

Reply:
Made the change